Cabal/src/Distribution/Simple/GHC.hs

```diff
@@ -163,14 +163,17 @@ configure verbosity hcPath hcPkgPath conf0 = do
       (userMaybeSpecifyPath "ghc" hcPath conf0)
   let implInfo = ghcVersionImplInfo ghcVersion
 
-  -- Cabal currently supports ghc >= 7.0.1 && < 9.12
+  -- Cabal currently supports GHC up to `maxGhcVersion`
```

Comment:
Maybe

```suggestion
  -- Cabal currently supports GHC up to `maxGhcVersion` not included.
```

Just to make sure it is clear we are talking about an open interval.

Reply:
That should be "inclusive", not "included".

Reply:
"Exclusive" maybe?

Reply:
I considered that, but it's potentially ambiguous to an ESL speaker as it is sometimes used to mean something roughly opposite to what we intend (think "an exclusive opportunity" and such), whereas the usage here is inspired by its mathematical meaning.

Reply:
I'd say "Cabal currently supports GHC < `maxGhcVersion`".

Reply:
I'm not a fan of mixing speech and symbols (there's a lot of advice against it in academic literature). But I hope "less than" is good enough.